mkdocs.yml

```diff
@@ -45,6 +45,7 @@ nav:
                          - 'Packaging and Delivering Software with IPS': dev/pdf/ips-dev-guide.pdf
                          - 'Using distribution constructor': dev/distribution-constructor.md
                          - 'Existing tasks': dev/existing-tasks.md
+                - 'Squeak on OpenIndiana': handbook/squeak.md
```

Comment:
Move links to http://docs.openindiana.org/handbook/community/. I don't think that it should be in the menu - it's too specific.